packages/uui-color-swatch/lib/uui-color-swatch.element.ts

```diff
@@ -143,6 +162,33 @@ export class UUIColorSwatchElement extends LabelMixin(
       .label {
         font-size: var(--uui-size-4);
       }
+
+      .color-swatch--big {
+        max-width: 120px;
+        width: 120px;
+        height: 50px;
+      }
+
+      .color-swatch__label {
+        width: 120px;
+        max-width: 120px;
```

Comment:
since width is already set, I would think its unnessecary to add the max-width. Sorry If I'm mistaken.

Reply:
max-width is important for the overflow, but seems like removing width is fine :)

Reply:
Good :-)